### PR TITLE
Use FILE and LINE macros for bug message

### PR DESCRIPTION
**Pull request by matsumotory**
@udzura I use __FILE__ and __LINE__ macros for code change!

### CODE REVIEW

File: src/mrb_mount.c [hunk 1]
```diff
@@ -42,7 +42,8 @@ static mrb_value mrb_mount_mount(mrb_state *mrb, mrb_value self)
     char buf[1024];
     char *ret;
     if ((ret = strerror_r(errno, buf, 1024)) == NULL) {
-      mrb_sys_fail(mrb, "[BUG] strerror_r failed at mrb_mount.c:44. Please report haconiwa-dev");
+      asprintf(&err_msg, "[BUG] strerror_r failed at %s:%d. Please report haconiwa-dev", __FILE__, __LINE__);
```
Comment (udzura) [inline src/mrb_mount.c:45]: Thanks! I just wanted to know how to use this kind of macros!!

Comment (matsumotory) [reply src/mrb_mount.c:45]: - if we write the following code

```c
 38   mrb_get_args(mrb, "zzz!iz!", &source, &target, &fstype, &mountflag, &data);
 39 
 40   asprintf(&err_msg, "[BUG] strerror_r failed at %s:%d. Please report haconiwa-dev", __FILE__, __LINE__);
 41   mrb_sys_fail(mrb, err_msg);
```

- output error 

```
RuntimeError: Mount#bind_mount => [BUG] strerror_r failed at /home/admin/matsumotory/work/mruby-mount/src/mrb_mount.c:40. Please report haconiwa-dev (mrbgems: mruby-mount)
RuntimeError: Mount#umount => [BUG] strerror_r failed at /home/admin/matsumotory/work/mruby-mount/src/mrb_mount.c:40. Please report haconiwa-dev (mrbgems: mruby-mount)
RuntimeError: Mount#mount => [BUG] strerror_r failed at /home/admin/matsumotory/work/mruby-mount/src/mrb_mount.c:40. Please report haconiwa-dev (mrbgems: mruby-mount)
```

Comment (matsumotory) [reply src/mrb_mount.c:45]: ref: https://gcc.gnu.org/onlinedocs/cpp/Standard-Predefined-Macros.html

Comment (udzura) [reply src/mrb_mount.c:45]: Thanks information!

**Pull request merged**